Cargo.toml

```diff
@@ -9,6 +9,16 @@ repository = "https://github.com/rust-lang/rustdoc-types"
 
 [dependencies]
 serde = {version="1", features=["derive"]}
+rustc-hash = {version="2", optional=true}
+
+[features]
+default = []
+
+# Switch the hashmaps used in rustdoc-types to the FxHashMap from rustc-hash.
```

Comment:
I think these documentation should go in either the README.md, or the crate-level doc comment (or both)

Reply:
I think all three! Because the docs.rs feature flags tab is a great place and should be used more c:

E.g. [clap's](https://docs.rs/crate/clap/latest/features)

Reply:
Yeah. it'll need to be in rust-lang/rust, but I don't think it's worth blocking this PR on that being done.